storybook/README.md

```diff
@@ -0,0 +1,82 @@
+# Building Storybook with Webassembly and Qt 5.14
+
+
+
+## Configuring the environment
+### Install Emscripten v1.38.27
+	
+
+    # Get the emsdk repo
+    git clone https://github.com/emscripten-core/emsdk.git
+    
+    #go to emsdk folder
+    cd emsdk
+    
+    #install Emscripten v1.38.27
+    ./emsdk install emscripten-1.38.27
+    
+    #activate emscripten-1.38.27
+    ./emsdk activate emscripten-1.38.27
+    
+    #install Fastcomp backend
+    ./emsdk install fastcomp-clang-tag-e1.38.27-64bit
+    
+    #activate Fastcomp backend
+    ./emsdk activate fastcomp-clang-tag-e1.38.27-64bit
+    
+    #add emsdk tools to env variables
+    #follow instructions received from activate command
+    #WARNING: this will configure the environment to use the emsdk compiler
+    #Ex:"which clang" command will now point to the emscripten clang
+    #To disable the env configuration comment the source command added in ~/.zprofile
```

Comment:
I would leave the warning because if the environment is configured to use emsdk tools, make won't work anymore for status-desktop. It will try to use the emsdk clang.

But I could better explain what following instructions received from activate command will do and it will be clearer why commenting the source command from ~/.zprofile is a good option to restore the env variables.

Reply:
Ok! I agree on adding a better explanaiton and then keeping this waring!! Thankss

Reply:
Done this is how it looks now:

     #add emsdk tools to env variables
     #this can be done by following instructions received from previous activate command
     #there are two options:

     #1. Configure the env variables for the current shell only:
     source emsdk_env.sh

     #2. Configure the env variables using the shell startup script:
     echo 'source "[path to emsdk folder]/emsdk_env.sh"' >> $HOME/.zprofile

     #WARNING: this will configure the environment to use the emsdk compiler
     #Ex:"which clang" command will now point to the emscripten clang instead of the system clang
     #to disable the env configuration comment the source command added earlier in ~/.zprofile

     #check environment
     #python needs to be installed. The emsdk scripts state that it should work with pyton 2 and 3
     #make sure python command can be resolved
     which python